Common/src/main/resources/assets/biomeswevegone/lang/ru_ru.json

```
@@ -467,10 +488,10 @@
   "block.biomeswevegone.mossy_red_rock_brick_stairs": "Ступени из мшистого красного камня",
   "block.biomeswevegone.mossy_red_rock_brick_wall": "Стена из мшистого красного камня",
   "block.biomeswevegone.mossy_red_rock_bricks": "Мшистый красный камень",
+  "block.biomeswevegone.mossy_stone": "Мшистый камень",
```

Для согласованности с офф. переводом всё "мшистое" должно стать "замшелым"
```suggestion
  "block.biomeswevegone.mossy_stone": "Замшелый камень",
```

---

"Mossy" для камней всегда "Замшелый"